server/graphql/mutations/expenses.js

```diff
@@ -33,6 +33,24 @@ async function _createActivity(expense, type) {
   });
 }
 
+async function _addUserIdToHostW9ReceivedList(expense) {
+  const host = await expense.collective.getHostCollective();
+  // If user is already included in Host data List, don't do anything
+  if (get(host, 'data.W9.receivedFromUserIds') && host.data.W9.receivedFromUserIds.includes(expense.UserId)) {
+    return false;
+  }
+  // Only Inserts User in W9 Received list if he is already present on
+  // the W9.requestSentToUserIds (which means this user already overstepped the W9 Threshold)
+  if (get(host, 'data.W9.requestSentToUserIds') && host.data.W9.requestSentToUserIds.includes(expense.UserId)) {
+    const receivedFromUserIds = !get(host, 'data.W9.receivedFromUserIds')
+      ? [expense.UserId]
+      : get(host, 'data.W9.receivedFromUserIds').concat([expense.UserId]);
```

Comment:
why `concat` and not `push`?

Reply:
This way is a "one line" solution. using push would require me at least 2 lines:

```
const receivedFromUserIds = !get(host, 'data.W9.receivedFromUserIds')
      ? []
      : get(host, 'data.W9.receivedFromUserIds');
receivedFromUserIds.push(expense.UserId);
```

But it's not a big deal, can do this way as well.